mono/mini/interp/mintops.def

```diff
@@ -22,6 +22,9 @@ OPDEF(MINT_POP1, "pop1", 1, Pop2, Push1, MintOpNoArgs)
 OPDEF(MINT_RET, "ret", 1, Pop1, Push0, MintOpNoArgs)
 OPDEF(MINT_RET_VOID, "ret.void", 1, Pop0, Push0, MintOpNoArgs)
 OPDEF(MINT_RET_VT, "ret.vt", 3, Pop1, Push0, MintOpInt)
+OPDEF(MINT_RET_LOCALLOC, "ret.localloc", 1, Pop1, Push0, MintOpNoArgs)
```

Comment:
Could use a comment stating that these should be in the same order as the MINT_RET opcodes.

Reply:
I will add it in future PR, if there aren't any other requested changes.